config/_peers/mainnet

```diff
@@ -3,3 +3,5 @@
 /dns4/keep-validator-0.prod-eks-eu-west-1.staked.cloud/tcp/3919/ipfs/16Uiu2HAm6Fs6Fn71n7PqRmpHMbfMkZUCGYhW5RL81MSMg57AANkZ
 /dns4/keep-validator-1.prod-eks-ap-northeast-2.staked.cloud/tcp/3919/ipfs/16Uiu2HAm5UzZb1TTYBjb2959h4z4VHzjt585SQqZnJPBrDnJuob7
 /dns4/keep-validator-2.prod-eks-eu-north-1.staked.cloud/tcp/3919/ipfs/16Uiu2HAmJvbYNhzY6a8kiG2zzrqXGnYWax7CQTbiMHoAvY4qLvg7
+/dns4/bootstrap-b1.threshold.p2p.org/tcp/3919/ipfs/16Uiu2HAmG7kjmp4D3C9jkvA8s59tfJzPfJSVqQJkhG6tEvVhJW68
+/dns4/bootstrap-b2.threshold.p2p.org/tcp/3919/ipfs/16Uiu2HAmSuXryEFqeDaMmRZ7KeUdQmHCt3PiWVNpe37q2xR3VT5n
```

Comment:
Hey @antonr-p2p, I think there is something off with `bootstrap-b2.threshold.p2p.org`. Can you double-check, please?

```
❯ telnet bootstrap-b1.threshold.p2p.org 3919
Trying 34.140.78.107...
Connected to bootstrap-b1.threshold.p2p.org.
Escape character is '^]'.
/multistream/1.0.0
^CConnection closed by foreign host.

❯ telnet bootstrap-b2.threshold.p2p.org 3919
Trying 35.228.156.5...
telnet: connect to address 35.228.156.5: Connection refused
telnet: Unable to connect to remote host
```

Reply:
@pdyraga fixed